Dockerfile

```diff
@@ -27,10 +27,13 @@ RUN chmod a+rwx $RUSTUP_DIR
 # Install Rust toolchain.
 ARG RUST_VERSION=1.36.0
 RUN rustup toolchain install $RUST_VERSION
+# Also install the nightly toolchain as it may be needed by some examples.
+RUN rustup toolchain install nightly
```

Comment:
But.. but.. reproducible builds and all that :)

Reply:
Is there a release scheduled/foreseeable of rust compiler with the features that require nightly? 🤔